tensorflow_quantum/core/qsim/state_space_test.cc

```diff
@@ -18,15 +18,380 @@ limitations under the License.
 #include <cmath>
 #include <complex>
 #include <memory>
+#include <string>
 
 #include "gtest/gtest.h"
+#include "tensorflow/core/lib/core/status.h"
+#include "tensorflow_quantum/core/proto/pauli_sum.pb.h"
 #include "tensorflow_quantum/core/qsim/mux.h"
+#include "tensorflow_quantum/core/src/circuit.h"
+
+#ifdef __AVX2__
+tfq::qsim::StateSpaceType STATE_SPACE_TYPE = tfq::qsim::StateSpaceType::AVX;
+#elif __SSE4_1__
+tfq::qsim::StateSpaceType STATE_SPACE_TYPE = tfq::qsim::StateSpaceType::SSE;
+#else
+tfq::qsim::StateSpaceType STATE_SPACE_TYPE = tfq::qsim::StateSpaceType::SLOW;
+#endif
```

Comment:
Since these are only used in the first test, what do you think about moving them inside of there and not making a global variable ?

Reply:
Moved inside of test